client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -622,11 +623,11 @@ const PersonalInfoTab: React.FC<Props> = ({ id }: Props): JSX.Element => {
                                                                     onChange={(event) => props.onChange(event.target.value)}
                                                                 >
                                                                     {
-                                                                        grades.map((grade: string) => (
+                                                                        educationGrades.map && educationGrades.map((grade: EducationGrade) => (
```

Comment:
what are you checking in educationGrades.map? maybe you meant  educationGrades && ... ?

Reply:
Because whenever the query isnt commited yet. it put on this varibale an empty object, so in order it not to crush I checked if the function exists first